lib/check_elastic_search.rb

```diff
@@ -3,14 +3,10 @@ module CheckElasticSearch
   def create_elasticsearch_doc_bg(_options)
     doc_id = Base64.encode64("#{self.class.name}/#{self.id}")
     return if doc_exists?(doc_id)
-    p = self.project
     ms = MediaSearch.new
     ms.id = doc_id
-    ms.team_id = self.team_id if self.is_a?(ProjectMedia)
-    unless p.nil?
-      ms.team_id = p.team_id
-      ms.project_id = [p.id]
-    end
+    ms.team_id = self.team_id
+    ms.project_id = self.project_ids
```

Comment:
do we need to update the ES schema to make sure `ms.project_id` can be a list?

Reply:
no `project_id` is already Array of integer